cosign.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: cosign
+  version: 1.13.1
+  description: Container Signing
+  target-architecture:

```

Comment:
Please add `epoch: 0` here, even if it is moot, it makes it easier to bump the epoch later if it is already present.